package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "karma-tap",
-  "version": "2.0.1",
-  "description": "adapter from karma to the tape testing framework",
+  "version": "2.1.0",
+  "description": "Karma Adapter to tape testing framework",
   "main": "index.js",
   "scripts": {
     "clean": "rimraf lib",
     "build:parser": "browserify src/parser.js --s tapParser -o lib/parser.js",
     "build-npm": "npm run clean && mkdirp lib && npm run build:parser",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "karma": "karma start",
+    "lint": "eslint . --ext .js,.jsx",
```

Comment:
This project doesn't include any es6 or jsx code, right? This line and the choice of eslint-standard-deviation seems off in that respect; including es6 code would pass lint but fail when it actually runs